doc/environment.yml

```diff
@@ -1,15 +1,25 @@
 name: test_env_xgcm_docs
+channels:
+  - conda-forge
+  - defaults
+  - anaconda
 dependencies:
   - python=3.9
-  - numba
   - pandoc
   - pip
+  - numba
+  - llvmlite
```

Comment:
I think lllvmlite is a dependency of numba and should be automatically pulled when installing numba (via conda at least)

Reply:
> I think lllvmlite is a dependency of numba and should be automatically pulled when installing numba (via conda at least)

That's true and it has been happening implicitly for all the previous successful builds. Ill give it a try again now that ive added `conda-forge`